types/index.d.ts

```diff
@@ -74,6 +74,7 @@ declare module 'highlight.js' {
         HASH_COMMENT_MODE: Mode
         NUMBER_MODE: Mode
         C_NUMBER_MODE: Mode
+        EXTENDED_NUMBER_MODE: Mode
```

Comment:
Dead code?

Reply:
Well, the code passes tests without that line, so I guess so. I was just following what seemed to be an established pattern, thinking everything like that should be declared there.